Lib/idlelib/config_key.py

```diff
@@ -219,33 +224,34 @@ def TranslateKey(self, key, modifiers):
         return key
 
     def OK(self, event=None):
-        if self.advanced or self.KeysOK():  # doesn't check advanced string yet
-            self.result=self.keyString.get()
+        key = self.keyString.get()
+        key.strip()
```

Comment:
`key.strip()` isn't inplace, so without assignment, it's not doing anything?

If that's the bug, it existed before, but wouldn't have been reached because the only way to enter whitespace would have been on the 'advanced' tab and this check was in `KeysOK`, which was never run when the advanced flag was set.

Thank you for giving me time to look at this and figure it out.